screensavers.rsxs/addon.xml.in

```diff
@@ -3,7 +3,7 @@
 debian package build.-->
 <addon
   id="screensavers.rsxs"
-  version="3.0.0"
+  version="3.0.1"
```

Comment:
Why bump versions to `3.0.1`?
`3.0.0` wasn't released yet.

Reply:
I can remove, in the past was only come on other places the question "why is version not increased" 😏